src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(callback, operatingValue) {
+      this.result = callback(operatingValue, this.result);
+
+      return this;
+    },
+    add(operatingValue, previousValue) {
+      return previousValue + operatingValue;
+    },
```

Comment:
Add blank lines between methods, it will be easier to read when they are visually separated